diff --git a/desktop/menus/view-menu.js b/desktop/menus/view-menu.js
index 8e23b5a5..eb69308e 100644
--- a/desktop/menus/view-menu.js
+++ b/desktop/menus/view-menu.js
@@ -6,12 +6,10 @@ const buildViewMenu = (settings, isAuthenticated) => {
   isAuthenticated = isAuthenticated || false;
 
   const themeSubMenu = [];
-  if (!platform.isLinux()) {
-    themeSubMenu.push({
-      label: '&System',
-      id: 'system',
-    });
-  }
+  themeSubMenu.push({
+    label: '&System',
+    id: 'system',
+  });
   themeSubMenu.push({
     label: '&Light',
     id: 'light',
